dotcom-rendering/src/components/ExpandableMarketingCard.tsx

```diff
@@ -0,0 +1,258 @@
+import type { SerializedStyles } from '@emotion/react';
+import { css } from '@emotion/react';
+import {
+	from,
+	headlineBold17,
+	headlineBold20,
+	neutral,
+	palette as sourcePalette,
+	space,
+	textSans15,
+	textSansBold12,
+	textSansBold14,
+} from '@guardian/source/foundations';
+import {
+	LinkButton,
+	SvgChevronDownSingle,
+	SvgCross,
+} from '@guardian/source/react-components';
+import type { Dispatch, SetStateAction } from 'react';
+import { getZIndex } from '../lib/getZIndex';
+import { palette } from '../palette';
+import { useConfig } from './ConfigContext';
+
+interface BannersLogoProps {
+	type: 'faded' | 'top' | 'bottom';
+	styles?: SerializedStyles;
+}
+
+const BannersLogo = ({ type, styles }: BannersLogoProps) => {
+	const { assetOrigin } = useConfig();
+	const src = `${assetOrigin}static/frontend/logos/red-blue-banner-${type}.svg`;
+	return <img src={src} alt="Guardian logo held up by hands" css={styles} />;
```

Comment:
These SVGs feel more decorative, rather than logos. I'd nuke the alt text as it's not adding anything useful,  interesting or [emotionally rich](https://tink.uk/text-descriptions-emotion-rich-images/). Arguably we could call these something other than `logos`... maybe illustrations, or maybe design has a better name for them 😊

```suggestion
	return <img src={src} alt="" css={styles} />;
```

Reply:
Ah thanks! This was mostly a copy-paste error 😅